src/utils/viewer/viewer.cpp

```diff
@@ -774,6 +778,11 @@ Viewer::Viewer(const Arguments& arguments)
               << simConfig_.sceneDatasetConfigFile
               << "| Loading Scene :" << simConfig_.activeSceneName;
 
+  // image based lighting (PBR)
+  if (args.isSet("ibl")) {
+    simConfig_.pbrImageBasedLighting = true;
+  }
```

Comment:
```suggestion
  simConfig_.pbrImageBasedLighting = args.isSet("ibl");
```

(sorry, had to make use of the opportunity :D)